samples/armv6-rpi-linux-gnueabihf/reported.by

```diff
@@ -0,0 +1,3 @@
+reporter_name="Yann E. MORIN"
```

Comment:
Stéphane,

I guess you should have changed that to your own identity! ;-)
(ditto for the URL, below).